src/Adapter/Module/PrestaTrust/ModuleEventSubscriber.php

```diff
@@ -77,4 +77,16 @@ public function onNewModule(ModuleZipManagementEvent $event)
         
         $this->checker->checkModuleZip($event->getModuleZip());
     }
+
+    /**
+     * Enable / disable the PrestaTrust feature
+     *
+     * @param bool $enable
+     * @return $this
+     */
+    public function toggleService($enable)
```

Comment:
Not sure toggle is the right name when you are not toggling but setting a value.

Reply:
I have been looking for a name used for enabling OR disabling in the same method. Having two different methods (enable() / disable()) wasn't convenient here.

Suggestions are appreciated. :)

Reply:
`setEnabled` & `isEnabled` for accessor?

Reply:
+1 for `setEnabled` / `isEnabled`

Reply:
+1 too

Reply:
or something like `configureActivation`, `setActivationStatus`

Reply:
well rather setActivated in that case since it's a boolean